testdata/networking/aosqe-pod-for-ping.json

```diff
@@ -10,7 +10,7 @@
   "spec": {
       "containers": [{
         "name": "hello-pod",
-        "image": "aosqe/pod-for-ping",
+        "image": "docker.io/aosqe/pod-for-ping@sha256:f61ca8fd1dd7dfb8d403b66df549ed5a94fc4626d3b19e2530af48aefa75e5ea",
```

Comment:
Prefix it with docker.io will not work in disconnected cluster even if you have mirror the image.

Reply:
are you sure? I think the `docker.io` is required. @wzheng1  please correct me if I'm wrong

Reply:
yes, docker.io is required when using imagecontentsourcepolicy in disconencted cluster. We need to add it to image mirror list and create ICSP for it. @xiuwang

Reply:
Ack